src/draw/nxp/vglite/lv_draw_vglite.c

```diff
@@ -118,7 +118,8 @@ void lv_draw_vglite_init(void)
     draw_vglite_unit->base_unit.name = "NXP_VGLITE";
 
 #if LV_USE_VGLITE_DRAW_THREAD
-    lv_thread_init(&draw_vglite_unit->thread, LV_THREAD_PRIO_HIGH, _vglite_render_thread_cb, 2 * 1024, draw_vglite_unit);
+    lv_thread_init(&draw_vglite_unit->thread, "vglite_th", LV_THREAD_PRIO_HIGHEST, _vglite_render_thread_cb, 2 * 1024,
```

Comment:
although here seems good, on a vglite MPU environment we will probably other threads with vglite naming, maybe put the LV prefix would be valid as well.